library/general/src/modules/ShadowConfig.rb

```diff
@@ -18,13 +18,14 @@
 # find current contact information at www.suse.com.
 
 require "yast"
-require "cfa/shadow_config"
+require "cfa/login_defs"
 
 module Yast
   # This class allows to access the API to handle login.defs attributes from Perl
   #
+  # In SLE 15 SP2, it relies on CFA::LoginDefs instead of CFA::ShadowConfig like later versions.
```

Comment:
The `CFA::ShadowConfig` is still  used in SP1  and GA, ritght? So the correct wording should be
```suggestion
  # In SLE 15 SP2, it relies on CFA::LoginDefs instead of CFA::ShadowConfig like in the earlier releases (SP1, GA)
```
I'd make it more clean.

Reply:
Ah, it seems I got it wrong, this looks correct, ignore this comment.